src/NServiceBus.Persistence.DynamoDB/Saga/SagaPersister.cs

```diff
@@ -311,5 +313,9 @@ public Task Complete(IContainSagaData sagaData, ISynchronizedStorageSession sess
             var sagaId = DynamoDBSagaIdGenerator.Generate(typeof(TSagaData), propertyName, propertyValue);
             return Get<TSagaData>(sagaId, session, context, cancellationToken);
         }
+
+        string SagaPartitionKey(Guid sagaId) => $"SAGA#{endpointIdentifier}#{sagaId}";
+
+        string SagaSortKey(Guid sagaId) => $"SAGA#{sagaId}";
```

Comment:
tbh we could simplify the sort key to not even contain the saga ID if we want to since we don't need the SK at all at the moment.

Reply:
I prefer to have it there. Better safe than sorry